python/py-asteval/Portfile

```diff
@@ -1,44 +1,42 @@
-                        a simple, safe, and robust miniature mathematical language that \
-                        can handle user-input. The emphasis here is on mathematical \
-                        expressions, and so many functions from numpy are imported and \
-                        used if available.
-platforms               darwin
-homepage                https://github.com/newville/asteval
-master_sites            pypi:a/asteval/
-distname                asteval-${version}
-
-checksums               rmd160  68782a598a9532d5384151796883fb42f1138b8a \
-                        sha256  38f3b0592cae7e7f65adc687e37aad1824a8e518245603a29ec33258277e779b \
-                        size    50686
-
-python.versions         27 34 35 36 37
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-asteval
+version             0.9.13
+categories-append   math
+license             BSD
+maintainers         {@reneeotten gmail.com:ottenr.work} openmaintainer
+description         Minimalistic evaluator of python expression using ast module
+long_description    ASTEVAL is a safe(ish) evaluator of Python expressions and \
+                    statements, using Python's ast module. The idea is to provide \
+                    a simple, safe, and robust miniature mathematical language that \
+                    can handle user-input. The emphasis here is on mathematical \
+                    expressions, and so many functions from numpy are imported and \
+                    used if available.
+platforms           darwin
+homepage            https://github.com/newville/asteval
+master_sites        pypi:a/asteval/
+distname            asteval-${version}
+
+checksums           rmd160  36f24c2c4509ba66e3a532686393b838e4ae12b4 \
+                    sha256  8faf9f92b2b0d2ed376dc149d6bab2e01f614f6994722be9c16f982cbdd07c99 \
+                    size    51640
+
+python.versions     27 34 35 36 37
 
 if {$subport ne $name} {
-    depends_build-append       port:py${python.version}-setuptools
+    depends_build-append    port:py${python.version}-setuptools
 
-    depends_lib-append         port:py${python.version}-numpy \
-                               port:py${python.version}-six
+    depends_lib-append      port:py${python.version}-numpy \
+                            port:py${python.version}-six
 
-    depends_test-append        port:py${python.version}-pytest
-    test.run                   yes
-    test.cmd                   py.test-${python.branch}
+    depends_test-append     port:py${python.version}-pytest
+    test.run                yes
+    test.cmd                py.test-${python.branch}
     test.target
-    test.env                   PYTHONPATH=${worksrcpath}/build/lib
+    test.env                PYTHONPATH=${worksrcpath}/build/lib
 
-    livecheck.type      none
-} else {
-    livecheck.type      pypi
+    livecheck.type          none
```

Comment:
This doesn't do the default check I think, instead, this disables the check, doesn't it?

Reply:
this does work as intended. It sets the livecheck to none within the ```if {${subport} ne ${name}}``` block (I just pushed a update commit where I added the curly-brackets...). Outside of this (i.e., when using ```py-asteval```) it uses the default PyPI livecheck. The else statement for the livecheck (present in many Portfiles) is redundant.